CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ endif()
 # compliler options
 set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_STANDARD_REQUIRED True)
+set(CMAKE_CXX_EXTENSIONS OFF)
```

Comment:
why?

Reply:
I was thought that there is a problem in cmake that fix with this one! 
I double checked again it is not needed!
IDK why it was needed the first time !